packages/amazonq/.changes/next-release/Bug Fix-ff32752d-fa0b-4d38-b719-68eb3c9ddca6.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "Q dev handle no change required"
```

Comment:
Thanks for including a changelog. Please review the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).